lib/counter/view/counter_page.dart

```diff
@@ -96,6 +97,9 @@ class _CounterTextState extends State<CounterText> {
       child: TextField(
         controller: _textController,
         style: theme.textTheme.displayLarge,
+        keyboardType: TextInputType.number,
+        inputFormatters: <TextInputFormatter>
+        [FilteringTextInputFormatter.digitsOnly,],
```

Comment:
Please reformat code in your IDE. Adding comma is the right thing to do. After reformatting IDE should automatically add new lines and spaces where it is needed.

For Android Studio please use Code->Reformat Code for tab that is in the focus